tests/scripts/all.sh

```diff
@@ -91,10 +91,11 @@
 set -eu
 
 pre_check_environment () {
-    if [ "$( uname )" != "Linux" ]; then
-        echo "This script only works in Linux" >&2
+    if uname -a | grep 'Linux.*x86_64' >/dev/null; then :; else
```

Comment:
I disagree with this change. I'd rather go in the other direction of making this script _more_ portable. We are Arm, after all. We should strive towards being able to develop on an Arm CPU. In the meantime, let's keep supporting 32-bit x86 environments where possible. And we should also strive towards making this script usable on OSX (if not Cygwin or WSL).